packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/components/DeleteButton.js

```diff
@@ -0,0 +1,84 @@
+import { Box, Button } from 'grommet'
+import { FormClose } from 'grommet-icons'
+import { bool, func, number, string } from 'prop-types'
+import styled from 'styled-components'
+
+import { useTranslation } from '@translations/i18n'
+
+const StyledButton = styled(Button)`
+  &:disabled {
+    cursor: not-allowed;
+  }
+
+  &:focus, &:hover {
+    svg {
+      border: 1px solid ${props => props.theme.global.colors['neutral-6']};
+      border-radius: 50%;
+    }
+  }
+`
+
+const IconWrapper = styled(Box)`
+  background: ${props => props.theme.global.colors.brand};
+  position: absolute;
+  border-radius: 50%;
+  transform: translate(-50%, -50%);
+  left: 50%;
+  top: 50%;
+  z-index: 2;
+`
+
+const ChildrenWrapper = styled(Box)`
+  position: relative;
+
+  & > img {
+    filter: blur(0.5px);
+  }
+
+  &::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: rgba(0, 93, 105, 0.6); // using neutral-1 #005D69 at 60%
+    pointer-events: none;
+    z-index: 1;
+  }
+`
+
+function DeleteButton ({
+  children,
+  choiceLabel = '',
+  deleteFn,
+  disabled = false,
+  tabIndex = -1
+}) {
+  const { t } = useTranslation()
+
+  return (
+    <StyledButton
```

Comment:
This needs some padding and margin when there is not a thumbnail image (see Figma).